modules/osd-create-cluster-gcp-account.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-a-gcp-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="osd-create-cluster-gcp-account_{context}"]
+= Creating a cluster on GCP with Google Cloud Marketplace
+
+When creating an {product-title} (OSD) cluster on Google Cloud through the OpenShift Cluster Manager Hybrid Cloud Console, customers can select Google Cloud Marketplace as their preferred billing model. This billing model allows Red Hat customers to take advantage of their link:https://cloud.google.com/docs/cuds[Google Committed Use Discounts (CUD)] towards {product-title} purchased through the Google Cloud Marketplace. Additionally, OSD pricing is consumption-based and customers are billed directly through their Google Cloud account.
+
+.Procedure
+
+. Log in to {cluster-manager-url} and click *Create cluster*.
+
+. In the *Cloud* tab, click *Create cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. Under *Billing model*, configure the subscription type and infrastructure type:
+.. Select the *On-Demand* subscription  type.
+.. From the dropdown menu, select *Google Cloud Marketplace*.
+.. Select the *Customer Cloud Subscription* infrastructure type.
+.. Click *Next*.
+. On the *Cloud provider* page, read the provided prerequisites and the Google terms and conditions. Add your service account key.
+.. Click the *Review Google Terms and Agreements* link.
+.. If you agree to the Google terms and agreements and want to proceed, continue completing the steps in the {product-title} cluster create wizard.
+.. Add your service account key.
++
+[NOTE]
+====
+For more information about service account keys, click the information icon located next to *Service account key*.
+====
+.. Click *Next*.
```

Comment:
It still needs to validate the sa. So as the same as the step 7 in [Creating a cluster on GCP with CCS](https://64901--docspreview.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster#osd-create-%5B%E2%80%A6%5Dster-on-gcp), `Click Next to validate your cloud provider account and go to the Cluster details page.` could be better.